### DISCUSSION TITLE
TypeError: query.getObserversCount is not a function

### DISCUSSION THREAD

**Discussion by ArmantoArisRoutsis** [TanStack/query#3716] [General]
Hello there,

I was recently migrating a project to react 18. I migrated everything successfully and there are no issues at all when I run the project. However when I go on the browser the app would not load and on the browser console I see the following error message:
![image](https://user-images.githubusercontent.com/67515158/174088222-9970e3b6-1c0f-49bd-bb49-c987a807487c.png)

It seems like the error has to do with the react query dev-tools as if I comment them out the project runs just fine with no issues at all.

`React` version: `18.1.0`
`react-query` version: `3.39.1`

**Comment by TkDodo** [comment]
please provide a codesandbox reproduction
